packages/management/addons-manager/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: addons-manager.tanzu.vmware.com.1.5.0
+  namespace: addons-manager
+spec:
+  refName: addons-manager.tanzu.vmware.com
+  version: 1.5.0
+  releaseNotes: "addons-manager 1.5.0"
+  licenses:
+    - "UNKNOWN"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tkg/packages/core/addons-manager@sha256:31c3f27350b693fec042ddead8906c119870ec8a876b9ee3e698a8d5d9c6d8b7
```

Comment:
this image url is also generated at the build time, so we can remove the hardcoded url

Reply:
removed the hardcoded url and changed to kbld-config.yaml variable